backend/clubs/views.py

```diff
@@ -1198,9 +1199,35 @@ def create(self, request, *args, **kwargs):
             raise DRFValidationError(
                 detail="Approved activities fair events have already been created. "
                 "See above for events to edit, and "
-                f"please email {settings.FROM_EMAIL} if this is en error."
+                "please email contact@pennclubs.com if this is en error."
```

Comment:
Change this back to `settings.FROM_EMAIL`.

The reason is that Hub@Penn has a different email address and we use the same backend. The error messages should have the right email to contact.